homeassistant/components/ntfy/notify.py

```diff
@@ -5,6 +5,7 @@
 from datetime import timedelta
 from typing import Any
 
+import aiofiles
```

Comment:
`aiofiles` is not a core dependency, thus meaning this becomes a requirement for this integration if this is imported (and thus should be added to the manifest).